README.md

```diff
@@ -100,15 +100,22 @@ library(ROI)
 library(ROI.plugin.glpk)
 library(ompr)
 library(ompr.roi)
-max_capacity <- 5
+max_capacity <- 3
 n <- 10
-weights <- runif(n, max = max_capacity)
-MIPModel() %>%
+fibs <- c(1, 1, 2, 3, 5, 8, 13, 21, 34, 55)
```

Comment:
Maybe add a small comment why fibonacci numbers are chosen? Otherwise people might spend a lot of time trying to understand the parameters instead of looking at the ompr model